duckdb-dataframe/benchmark.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Install
+
+sudo apt-get update
+sudo apt-get install -y python3-pip
+pip install pandas chdb
```

Comment:
I had to use `pip install --break-system-packages` here (as it complained without the flag). Interestingly, the `pip install` commands in other ClickBench scripts also don't use that flag, so I am okay with not having it (it might be some weirdness with my system).

Reply:
I also encountered the same problem and checked how other Python pkgs worked. I think this should be the Ubuntu 24 introduced new problem.
But after that I decided to leave it identical with other packages. Maybe we could fix all of them with another patch.

Reply:
https://github.com/ClickHouse/ClickBench/pull/223